docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md

```diff
@@ -35,18 +36,35 @@ AlmaLinux OS 8 disk images are used by default, but you can specify a custom dis
 
 YugabyteDB Anywhere requires the following additional software to be pre-installed on nodes:
 
-- Python 3.6-3.8
-- Install the python selinux package corresponding to your version of python. For example, using pip, you can install as follows:
-
-  `python3 -m pip install selinux`
-
-  Alternately, if you are using the default version of python3, you might be able to install the python3-libselinux package.
-
 - OpenSSH Server. Allowing SSH is recommended but optional. Using SSH can be skipped in some on-premises deployment approaches; all other workflows require it. [Tectia SSH](../../create-deployments/connect-to-universe/#enable-tectia-ssh) is also supported.
 - tar
 - unzip
 - policycoreutils-python-utils
 
+#### Python
+
+Python 3.8-3.9 must be installed on the nodes.
```

Comment:
For target node python can be Python 3.6 - 3.12. From https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix we are on ansible 9.2 which uses ansible-core 2.16 so those are the python requirements we have right now.

Reply:
cc @Vars-07

Reply:
But also this is only for master (so preview). All released versions we were on ansible-core 2.10 so the requirements were different.